plugin/src/main/kotlin/com/github/triplet/gradle/play/internal/Listings.kt

```diff
@@ -20,23 +23,28 @@ internal enum class ListingDetail(val fileName: String, val maxLength: Int = Int
     FULL_DESCRIPTION("fulldescription", 4000),
     VIDEO("video"),
 
-    WHATS_NEW("whatsnew", 500),
+    WHATS_NEW("whatsnew", 500)
+}
 
+internal enum class AppDetail(val fileName: String, val maxLength: Int = Int.MAX_VALUE) {
     CONTACT_EMAIL("contactEmail"),
     CONTACT_PHONE("contactPhone"),
     CONTACT_WEBSITE("contactWebsite"),
-    DEFAULT_LANGUAGE("defaultLanguage");
+    DEFAULT_LANGUAGE("defaultLanguage")
 }
 
-internal enum class ImageType(val fileName: String, val maxNum: Int = 8) {
-    ICON("icon", 1),
-    FEATURE_GRAPHIC("featureGraphic", 1),
-    PROMO_GRAPHIC("promoGraphic", 1),
+// Min length for any side: 320px. Max length for any side: 3840px.
+internal data class ImageSize(val minWidth: Int, val minHeight: Int, val maxWidth: Int = minWidth, val maxHeight: Int = minHeight)
```

Comment:
This is and the enum declaration below are too long. Should be like this:
```kt
internal data class ImageSize(
        val minWidth: Int,
        val minHeight: Int,
        val maxWidth: Int = minWidth,
        val maxHeight: Int = minHeight
)
```

Also, can we move the data class below the enum? Standard practice is to go `fields -> methods -> objects -> enums -> classes`.